README.md

```diff
@@ -1,83 +1,56 @@
-# Cxbx-Reloaded game-compatibility project
+# Cxbx-Reloaded game-compatibility
+This project is intended to gather all compatibility reports for [Cxbx-Reloaded](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded).
 
-This project is intended to gather all compatibility reports for Cxbx-Reloaded.
+*This project is NOT intended for issues that are emulation-related, or tech support.*
 
-(Cxbx-Reloaded is an original Xbox emulator, see [the Cxbx-Reloaded project on GitHub](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded).)
+*Emulation-related issues must be submitted to [the Cxbx-Reloaded project itself](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/issues).*
 
-
-*This project is NOT intended for issues that are emulation-related.*
-
-*(Emulation-related issues must be submitted to [the Cxbx-Reloaded project itself](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/issues).)*
+***Tech support belongs on the [Discord](https://discord.gg/26Xjx23).***
 
 ## Unique reports
-
 * All compatibility reports must be reported [here, in this issue tracker](https://github.com/Cxbx-Reloaded/game-compatibility/issues).
-* In this game-compatibility project, each title is assigned ONE issue.
-* Do not create duplicate issues (first, search if a title already has an issue).
-* Duplicate reports will be merged into one, duplicates will be closed.
-* Unique game-compatibility issues are kept open forever, regardless of state.
-
+* In this game-compatibility project, each title is assigned ONE issue per version.
+* Do not create duplicate issues. Search if a title already has an issue first.
+  * Duplicate reports will be merged into one, duplicates will be closed.
 
 ## Issue titles
-
 Issue titles MUST mention the XBE title, optionally followed by an alternative title in square brackets, followed by a mandatory Title ID in square brackets, then lastly followed by a mandatory version number in square brackets. Nothing else.
 
-Example : [FROM RUSSIA WITH LOVE (TM) \[007: From Russia with Love\] \[45410079\] \[1.01\]](https://github.com/Cxbx-Reloaded/game-compatibility/issues/10).
-
-Verify titles against [the Xbox-Games database](http://www.xbox-games.org/) and [this Xbox1 name lookup](https://github.com/roman5566/i360gm/blob/master/Xbox1NameLookup.csv) list.
+Example: [FROM RUSSIA WITH LOVE (TM) \[007: From Russia with Love\] \[45410079\] \[1.01\]](https://github.com/Cxbx-Reloaded/game-compatibility/issues/10).
 
+Verify titles against [the Xbox-Games database](https://www.xbox-games.org) and [this Xbox1 name lookup](https://github.com/roman5566/i360gm/blob/master/Xbox1NameLookup.csv) list.
```

Comment:
The best source is redump's database, but for some reason they no longer list Version numbers on their website. 

This spreadsheet may be a good source (it is seeded from redump data).
https://docs.google.com/spreadsheets/d/1e33LTPD6TCuLqszv6orQzzlkHRFuSnIlk-OQWn5M2y8/edit#gid=347627117